frontend/src/views/ProducerForm/ProductStep.vue

```diff
@@ -0,0 +1,146 @@
+        <DsfrInput :model="payload.gamme" label-visible label="Gamme" />
+      </DsfrInputGroup>
+
+      <!-- Useless? -->
+      <DsfrInputGroup>
+        <DsfrInput :model="payload.flavor" label-visible label="Arôme" />
+      </DsfrInputGroup>
+    </div>
+  </div>
+  <h2 class="fr-h6 !mt-8">
+    <v-icon class="mr-1" name="ri-capsule-fill" />
+    Format
+  </h2>
+  <DsfrInputGroup class="mt-6 max-w-md">
+    <DsfrSelect label="Forme galénique" :model="payload.dosageForms" :options="dosageForms" :required="true" />
+  </DsfrInputGroup>
+  <DsfrFieldset legend="Poids ou volume d'une unité de consommation" legendClass="fr-label !font-normal !pb-0">
+    <div class="flex">
+      <div class="max-w-64">
+        <DsfrInput :model="payload.unitQuantity" class="max-w-64" :required="true" />
+      </div>
+      <div class="max-w-32 ml-4">
+        <DsfrSelect :options="units" :model="payload.unitMeasurement" defaultUnselectedText="Unité" />
+      </div>
+    </div>
+  </DsfrFieldset>
+  <div class="grid grid-cols-2 gap-4">
+    <div class="col-span-2 md:col-span-1 max-w-md">
+      <DsfrInputGroup>
+        <DsfrInput
+          :model="payload.dailyRecommendedDose"
+          label-visible
+          label="Dose journalière recommandée"
+          :required="true"
+        />
+      </DsfrInputGroup>
+    </div>
+    <div class="col-span-2 md:col-span-1 max-w-md">
+      <DsfrInputGroup>
+        <DsfrInput
+          :model="payload.minimumDuration"
+          label-visible
+          label="Durabilité minimale / DLUO (en mois)"
+          :required="true"
+        />
+      </DsfrInputGroup>
+    </div>
+  </div>
+  <h2 class="fr-h6 !mt-8">
+    <v-icon class="mr-1" name="ri-file-user-fill" />
+    Populations cible
+  </h2>
+  <DsfrFieldset legend="Population cible" legendClass="fr-label">
+    <div class="grid grid-cols-6 gap-2">
+      <DsfrCheckbox
+        v-for="population in populations"
+        :model="payload.targetPopulations"
+        :key="`population-${population.id}`"
+        :label="population.name"
+        class="col-span-6 sm:col-span-3 lg:col-span-2"
+      />
+    </div>
+  </DsfrFieldset>
+
+  <DsfrFieldset legend="Consommation déconseillée" legendClass="fr-label">
+    <div class="grid grid-cols-6 gap-2">
+      <DsfrCheckbox
+        v-for="condition in conditions"
+        :model="payload.targetConditions"
+        :key="`condition-${condition.id}`"
+        :label="condition.name"
+        class="col-span-6 sm:col-span-3 lg:col-span-2"
+      />
+    </div>
+  </DsfrFieldset>
+  <h2 class="fr-h6 !mt-8">
+    <v-icon class="mr-1" name="ri-focus-2-fill" />
+    Objectifs / effets
+  </h2>
+  <h2 class="fr-h6 !mt-8">
+    <v-icon class="mr-1" name="ri-home-2-fill" />
+    Adresse sur l'étiquetage
+  </h2>
+  TODO car on aura déjà l'adresse à partir du SIRET
+</template>
+<script setup>
+import { ref, onMounted } from "vue"
+import { verifyResponse } from "@/utils"
+
+const payload = ref({})
+const dosageForms = [
+  {
```

Comment:
Dans cette issue je liste les tables qu'on sait déjà qu'il nous manque. J'y ajoute les formes galeniques du coup https://github.com/orgs/betagouv/projects/67/views/1?pane=issue&itemId=52499888

Reply:
`galenicFormulation` in english btw

Reply:
Thank you :) 
Fix : https://github.com/betagouv/complements-alimentaires/pull/153/commits/3534cbde966526c4d245aa5df01b7472d2fd1b45